### PR TITLE
fix navbar

### LINKED ISSUE #109: Display username on nav bar
## Expected Behavior
Once a user is logged in, display its username on the right side of the navbar, with the 'logout' button to the left/right.

## Actual Behavior
The username isn't displayed, and the logout button aligned to the left of the navbar.

### PR DESCRIPTION
**Pull request by avraham1992**
Fixes #109 #115 

## Changes you made

-fix the nav bar 
-add user name after you login to the navbar

### CODE REVIEW

**Review by orzionpour — CHANGES_REQUESTED** [review]
Maybe it's only on my local repo, but the username isn't displayed on the navbar.

**Comment by avraham1992** [comment]
> Maybe it's only on my local repo, but the username isn't displayed on the navbar.
its only your... its work i check now agen bro

**Comment by avraham1992** [comment]
> Maybe it's only on my local repo, but the username isn't displayed on the navbar.
 Apparently I made the two changes separately and uploaded reg I changed now

File: templates/navbar.html [hunk 1]
```diff
@@ -15,19 +15,20 @@
...
       </div>
+    </div>
       <!-- Navbar Right Side -->
       <div class="navbar-nav">
         {% if user.user.is_authenticated %}
-          <a class="nav-item nav-link" href="{% url 'tasks' %}">View Tasks</a>
-            {% if user.is_manager %}
-                <a class="nav-item nav-link" href={% url 'new_task' %}>New Task</a>
-            {% endif %}
+            <a class="nav-item nav-link" href="#">{{ user.user.username }}</a>
```
Comment (orzionpour) [inline templates/navbar.html:27]: As for now there is no redirection here, so maybe the "href" should be ommitted.
```suggestion
            <a class="nav-item nav-link">{{ user.user.username }}</a>
```

**Review by orzionpour — COMMENTED** [review]
Now it works, looks great !
Consider changing this line.

**Review by orzionpour — DISMISSED** [review]

File: templates/navbar.html (hunk 1 above)
Comment (avraham1992) [reply templates/navbar.html:27]: @orzionpour Trust me this is a preparation for a pizzeria that I will do ... to a personal profile page for each user)

**Review by aradz1997 — APPROVED** [review]

**Comment by kobihk** [comment]
> * **Please mention someone for code review**

please remove the last line from the PR description

File: templates/navbar.html (hunk 1 above)
Comment (kobihk) [reply templates/navbar.html:27]: @avraham1992 if this code is not relevant for now, I think you can remove it

**Review by kobihk — CHANGES_REQUESTED** [review]

**Comment by kobihk** [comment]
@avraham1992 please remove the last line:
"Please mention someone for code review"
from the PR description.

**Comment by avraham1992** [comment]
> @avraham1992 please remove the last line: "Please mention someone for code review" from the PR description.

@kobihk done

**Review by kobihk — APPROVED** [review]

**Review by AvivLiberman — APPROVED** [review]
Checked it locally looks great, nice job!

**Review by Asafh7 — APPROVED** [review]

**Pull request merged**